pkg/omf/cli/omf.update.fish

```diff
@@ -15,14 +22,19 @@ function omf.update -a name
     set parent_path "pkg"
   end
 
-  for path in {$OMF_PATH,$OMF_CONFIG}/pkg/$name
+  set update_status error
+  for path in {$OMF_PATH,$OMF_CONFIG}/$parent_path/$name
     not test -e "$path/.git"; and continue
 
-    omf.repo.pull $path; and set return_success
-
-    set -q return_success;
-      and omf.bundle.install $path/bundle
+    omf.repo.pull $path
+    switch $status
+    case 304
```

Comment:
Shell ain't HTTP. I would prefer an approach where the function return one of:

+ `return -1`: Nothing to do
+ `return  0`: Updated
+ `return  1`: Error updating